ui.apps/src/main/content/jcr_root/apps/wcm/core/content/editor/_jcr_content/content/items/content/header/items/acs-commons_copy-publish-url-dialog/.content.xml

```diff
@@ -0,0 +1,53 @@
+<!--
+  ~ ACS AEM Commons
+  ~
+  ~ Copyright (C) 2013 - 2024 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0"
+          xmlns:jcr="http://www.jcp.org/jcr/1.0"
+          xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          granite:id="acs-aem-commons__aem-sites-copy-publish-url"
+          jcr:primaryType="nt:unstructured"
+          jcr:title="Publish URLs"
+          sling:resourceType="granite/ui/components/coral/foundation/dialog"
+          closable="{Boolean}true">
+    <granite:data
+            jcr:primaryType="nt:unstructured"
+            assetpath="${empty requestPathInfo.suffix ? paramValues[&quot;item&quot;][0] : requestPathInfo.suffix}"/>
+    <items jcr:primaryType="nt:unstructured">
+        <clientlibs
+                jcr:primaryType="nt:unstructured"
+                sling:resourceType="granite/ui/components/coral/foundation/includeclientlibs"
+                js="[acs-commons.authoring.sites.copy-publish-url]">
+            <granite:rendercondition
+                    jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/renderconditions/and">
+                <condition1
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/coral/foundation/renderconditions/simple"
+                        expression="${empty paramValues['item'][1] ? 'true' : 'false'}"/>
+                <condition2
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="acs-commons/renderconditons/ispublished"
+                        resourcePath="${requestPathInfo.suffix}"/>
+                <condition3
+                        jcr:primaryType="nt:unstructured"
+                        sling:resourceType="granite/ui/components/foundation/renderconditions/feature"
+                        feature="com.adobe.acs.commons.wcm.impl.copysitespublishurlfeature.feature.flag"/>
+            </granite:rendercondition>
+        </clientlibs>
+    </items>
```

Comment:
Conditionally loading the JS if this feature is enabled.

I dont know why, but the Page Editor seems to discard the clientlibs CSS when its loaded here, which is why only JS is loaded. In the JS file, the CSS is then loaded into the page. Beats me.

Reply:
Weird.